terraform/aks/terraform.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = "~> 1.4"
```

Comment:
From experience it's easier to pin the version exactly (=) for terraform as well as the providers to avoid surprises later on

Reply:
Yeah, no problem, I've pinned the versions of Terraform and the provider.